docs/supported_platforms.md

```diff
@@ -20,7 +20,25 @@ TODO document this platform
 
 ## Azure
 
-TODO document this platform
+File `osServicePrincipal.json` in `~/.azure/` directory with proper credentials
+is required. The file looks like this:
+
+```json
+{
+  "subscriptionId": "...",
+  "clientId": "...",
+  "clientSecret": "...",
+  "tenantId": "..."
+}
+```
+
+To deploy ocs-ci cluster on Azure platform using one of OCS QE subscriptions,
+you need to specify one deployment config file from `conf/deployment/azure/`
+directory (which one to use depends on the type of deployment one needs to
+use) and also one config file with azure `base_domain` value (either
+`conf/ocsci/azure_qe_rh_ocs_com_domain.yaml` or
+`conf/ocsci/azure2_qe_rh_ocs_com_domain.yaml` file), because each azure
+subscription has it's own domain name.
```

Comment:
I would also add some note if someone is using it outside of QE that they have to create custom config file similar to one of the mentioned above to define their custom base_domain.

Reply:
Updated in 89ec613.